x-pack/plugins/enterprise_search/public/assets/source_icons/native_connector_icons.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import mongodb from './mongodb.svg';
+import mysql from './mysql.svg';
+
+export const NATIVE_CONNECTOR_ICONS = {
+  mongodb,
+  mysql,
+} as { [key: string]: string };
```

Comment:
nitpick(non-blocking): 
```suggestion
export const NATIVE_CONNECTOR_ICONS: Record<string, string | undefined> = {
  mongodb,
  mysql,
};
```